src/BlockTypes/MiniCart.php

```diff
@@ -338,4 +403,53 @@ protected function get_markup( $attributes ) {
 			</div>
 		</div>';
 	}
+
+	/**
+	 * Return an instace of the CartController class.
+	 *
+	 * @return CartController CartController class instance.
+	 */
+	protected function get_cart_controller() {
+		return new CartController();
+	}
+
+	/**
+	 * Get array with data for handle the tax label.
+	 * the entire logic of this function is was taken from:
+	 * https://github.com/woocommerce/woocommerce/blob/e730f7463c25b50258e97bf56e31e9d7d3bc7ae7/includes/class-wc-cart.php#L1582
+	 *
+	 * @return array;
+	 */
+	protected function get_tax_label() {
+		$cart = WC()->cart;
+
+		if ( $cart->display_prices_including_tax() ) {
+			if ( ! wc_prices_include_tax() ) {
+				$label_including_tax               = WC()->countries->inc_tax_or_vat();
+				$display_cart_prices_including_tax = true;
+				return array(
+					'label_including_tax'               => $label_including_tax,
+					'display_cart_prices_including_tax' => $display_cart_prices_including_tax,
+				);
+			}
+			return array(
+				'label_including_tax'               => '',
+				'display_cart_prices_including_tax' => true,
+			);
+		}
+
+		if ( wc_prices_include_tax() ) {
+			$label_including_tax               = WC()->countries->ex_tax_or_vat();
+			$display_cart_prices_including_tax = true;
+			return array(
+				'label_including_tax'               => $label_including_tax,
+				'display_cart_prices_including_tax' => false,
```

Comment:
This is a bit confusing, we set `$display_cart_prices_including_tax` to true, but never use it again inside this method. But then, it returns `'display_cart_prices_including_tax' => false`. Was that an overlook or am I missing something?

Reply:
I forgot to remove it